paths/categories/topics_count.yaml

```diff
@@ -5,21 +5,23 @@ get:
   description: |
     Allows to get detailed count of topics by category and type. Returns different responses for different clients because they may have different topic types enabled. Allows filtering by category ids
   parameters:
-    - name: categories[]
+    - name: categories
       in: query
       description: |
-        Ids of categories that should be returned. Multiple categories have to be requested like "?categories[]=1&categories[]=2&categories[]=3"
+        IDs of categories to count topics from
```

Comment:
the description included the definition with the `param[]` because it's not a standard. do we have a place where this can be looked up in this document, for external developers?

Reply:
No, we don't. I thought this was standard for form submissions? But yeah, might be better to leave it there since we're using request bodies too